cds_ils/config.py

```diff
@@ -631,3 +631,7 @@ def _parse_env_bool(var_name, default=None):
     "ebl": {"priority": 3, "agency_code": "MiAaPQ"},
     "safari": {"priority": 4, "agency_code": "CaSebORM"},
 }
+
+CDS_ILS_IMPORTER_UPLOADS_PATH = "/tmp"
+
+CDS_ILS_IMPORTER_FILE_EXTENSIONS_ALLOWED = ["xml"]
```

Comment:
Very minir: no `.` in the end? Otherwise I could probably upload something `.axml` :)

Reply:
Nice catch, you're totally right.